dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -17,33 +20,43 @@ class SupplierFrameworkData(object):
 
     data = None
 
-    def __init__(self, client, target_framework_slug):
+    def __init__(self, client, target_framework_slug, *, supplier_ids=None):
```

Comment:
:+1:  Neat use of `*` trick here, I should probably start using it.

For the uninitiated, this is a trick that causes, in this case, `supplier_ids` to only be specifiable as a kwarg, rather than have the odd ambiguity of it also being specifiable positionally.